src/strings.c

```diff
@@ -0,0 +1,1177 @@
+    _("{color YELLOW}Normal{reset}"),
+    _("{color YELLOW}Fire{reset}  "),
+    _("{color YELLOW}Water{reset}"),
+    _("{color YELLOW}Grass{reset}"),
+    _("{color YELLOW}Electric{reset}"),
+    _("{color YELLOW}Ice{reset}  "),
+    _("{color YELLOW}Fighting{reset}"),
+    _("{color YELLOW}Poison{reset}"),
+    _("{color YELLOW}Ground{reset}"),
+    _("{color YELLOW}Flying{reset}"),
+    _("{color YELLOW}Psychic{reset}"),
+    _("{color YELLOW}Bug{reset}"),
+    _("{color YELLOW}Rock{reset}"),
+    _("{color YELLOW}Ghost{reset}"),
+    _("{color YELLOW}Dragon{reset}"),
+    _("{color YELLOW}Dark{reset}"),
+    _("{color YELLOW}Steel{reset}"),
+};
+
+const u8 *const gUnknown_810AF50[] = {
+    [0] = _("Arrows"),
+    [1] = _("Rocks"),
+    [2] = _("Grass"),
+    [3] = _("Bread"),
+    [4] = _("Rings"),
+    [5] = _("Machine A"),
+    [6] = _("Money"),
+    [7] = _("Others"),
+    [8] = _("Original"),
+    [9] = _("Machine B"),
+    [10] = _("WG Machine"),
+    [11] = _("Used TM"),
+    [12] = _("Event A"),
+    [13] = _("Event B"),
+    [14] = _("Event C"),
+    [15] = _("Event D"),
+    [16] = _("Keys"),
+    [17] = _("Event E"),
+    [18] = _("Favorites"),
+    [19] = _("Drinks"),
+};
+
+const u8 *const gAdventureLogText[] = {
+    [0] = _("Reached the Hill of the Ancients."),
+    [1] = _("Took to the road as fugitives."),
+    [2] = _("Prevented the meteor's collision."),
+    [3] = _("Broke the curse on Gardevoir."),
+    [4] = _("The rescue team base was completed."),
+    [5] = _("Rescued Smeargle."),
+    [6] = _("Spotted Munchlax."),
+    [7] = _("{VALUE_0} moves were learned."),
+    [8] = _("{VALUE_0} Friend Rescues were successful."),
+    [9] = _("{VALUE_0} Pokémon evolved."),
+    [10] = _("{VALUE_0} Pokémon joined the team."),
+    [11] = _("{VALUE_0} thieving attempts succeeded."),
+    [12] = _("{VALUE_0} floors were explored."),
+    [13] = _("All Friend Areas were obtained."),
+    [14] = _("All Pokémon were made leaders."),
+    [15] = _("All Pokémon joined the team."),
+    [16] = _("Moltres joined the team."),
+    [17] = _("Zapdos joined the team."),
+    [18] = _("Articuno joined the team."),
+    [19] = _("Deoxys joined the team."),
+    [20] = _("Entei joined the team."),
+    [21] = _("Raikou joined the team."),
+    [22] = _("Suicune joined the team."),
+    [23] = _("Ho-Oh joined the team."),
+    [24] = _("Kyogre joined the team."),
+    [25] = _("Groudon joined the team."),
+    [26] = _("Rayquaza joined the team."),
+    [27] = _("Lugia joined the team."),
+    [28] = _("Celebi joined the team."),
+    [29] = _("Mew joined the team."),
+    [30] = _("Mewtwo joined the team."),
+    [31] = _("Jirachi joined the team."),
+};
+
+const u8 *const gRescueTeamRanks[] = {
+    [0] = _("Normal Rank"),
+    [1] = _("Bronze Rank"),
+    [2] = _("Silver Rank"),
+    [3] = _("Gold Rank"),
+    [4] = _("Platinum Rank"),
+    [5] = _("Diamond Rank"),
+    [6] = _("Lucario Rank"),
+};
+
+const u8 *const gAbilityNames[] = {
```

Comment:
Probably should create a `MAX_ABILITIES` in the header and use it to bound this one.